.azure/pipelines/ci.yml

```diff
@@ -659,6 +661,17 @@ stages:
         beforeBuild:
         - powershell: "& ./src/Servers/IIS/tools/UpdateIISExpressCertificate.ps1; & ./src/Servers/IIS/tools/update_schema.ps1"
           displayName: Setup IISExpress test certificates and schema
+        - task: Cache@2
+          inputs:
+            key: 'pw | "$(Agent.OS)" | eng/versions.props'
```

Comment:
eng/Versions.props doesn't list the `npm` package version for Playwright. I'm not sure where the version is stored. @javiercn❔ If we just use the latest, caching would be a **Bad Thing:tm:**.